lib/generateMarkdown.ts

```diff
@@ -112,13 +112,29 @@ export async function generateMarkdown(
 
   // Generate Review Table
   markdown += "## Review Table\n\n"
+  markdown += `
+[contributor-hover]: ## "Name of the contributor"
+[reviews-hover]: ## "Number of reviews received for PRs"
+[approvals-hover]: ## "Number of approvals received for PRs"
+[rejections-hover]: ## "Number of rejections received for PRs"
+[changes-hover]: ## "Number of change requests received"
+[prs-opened-hover]: ## "Number of PRs opened"
+[prs-closed-hover]: ## "Number of PRs closed"
+[issues-hover]: ## "Number of issues created by the contributor"
+[bountied-issues-hover]: ## "Number of issues with a bounty"
+[bountied-issue-amount-hover]: ## "Total bounty amount in dollars"
+`
   markdown +=
-    "| Contributor | Reviews Received | Approvals | Rejections | Changes Requested | PRs Opened | PRs Closed | Issues Created | Bountied Issues | Bountied Issue $ |\n"
+    "| [Contributor][contributor-hover] | [Reviews Received][reviews-hover] | [Approvals][approvals-hover] | [Rejections][rejections-hover] | [Changes Requested][changes-hover] | [PRs Opened][prs-opened-hover] | [PRs Closed][prs-closed-hover] | [Issues Created][issues-hover] | [Bountied Issues][bountied-issues-hover] | [Bountied Issue $][bountied-issue-amount-hover] |\n"
```

Comment:
```suggestion
    "| [Contributor][contributor-hover] | [Reviews Received][reviews-hover] | [Approvals][approvals-hover] | [Rejections][rejections-hover] | [Rejections Received][changes-hover] | [PRs Opened][prs-opened-hover] | [PRs Closed][prs-closed-hover] | [Issues Created][issues-hover] | [Bountied Issues][bountied-issues-hover] | [Bountied Issue $][bountied-issue-amount-hover] |\n"
```

Reply:
@divanshu-go i don't even think i fixed them properly. Keep in mind, me, @seveibar should have many rejections given and basically 0 rejections received. Can you fix these columns? I literally can't evaluate the table you're showing me in the comments so I can't approve in this PR because I don't know what the columns mean because the labels are still ambiguous

The tooltips are great, but I can't use them in this review so i still have no idea what the columns mean unless I do some cross referencing or something